modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
+* A network attachment definition is configured in the same namespace as your VM.
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. Start the VM to which you want to hot plug the network interface by using the following command:
++
+[source,terminal]
+----
+$ virtctl start <vm_name>
+----
++
+.Example VM configuration
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          interfaces:
+          - masquerade: {}
+            name: defaultnetwork
+          rng: {}
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: defaultnetwork
+        pod: {}
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: quay.io/kubevirt/fedora-with-test-tooling-container-disk:devel
+        name: containerdisk
+----
+
+. Use the following command to hot plug a new network interface to the running VM:
++
+[source,terminal]
+----
+$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+----
++
+where:
+
+<vm_name>:: Specifies the name of the `VirtualMachine` object.
```

Comment:
Please use callouts.

Reply:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output), using callouts would be impractical in this case because all three variables are on the same line.

Reply:
I do it all the time: "<1> Specify x, y, and z."

Actually, I'm not sure this even needs callouts because the flags are self-evident. 

Suggestion: "$ virtctl addinterface <vm_name> --network-attachment-definition-name <nad_name> --name <interface_name>" with no callouts.